libs/sql-parser/src/main/java/io/crate/sql/tree/AddColumnDefinition.java

```diff
@@ -43,29 +40,14 @@ public AddColumnDefinition(T name,
                                @Nullable T generatedExpression,
                                @Nullable ColumnType<T> type,
                                List<ColumnConstraint<T>> constraints) {
-        this(name, generatedExpression, type, constraints, true, generatedExpression != null);
-    }
-
-    public AddColumnDefinition(T name,
-                               @Nullable T generatedExpression,
-                               @Nullable ColumnType<T> type,
-                               List<ColumnConstraint<T>> constraints,
-                               boolean validate,
```

Comment:
@BaurzhanSakhariev Could you please double check that his flag is not needed, also for your current work with bug fixing and `_raw` optimization?

Reply:
Hi, it looks unrelated to _raw optimization.